create.raml

```diff
@@ -0,0 +1,75 @@
+#%RAML 0.8
+title: Create
+version: v0.1.1-20180620
+documentation:
+
+  - title: OCF Copyright
+    content: Copyright (c) 2018 Open Connectivity Foundation, Inc. All rights reserved.
+  - title: OCF License
+    content: !include LICENSE.md
+  - title: OCF Disclaimer
+    content: !include DISCLAIMER.md
+
+
+schemas:
+  - create: !include schemas/oic.if-create-schema.json
+  - create-response: !include schemas/oic.if-create-response-schema.json
+
+traits:
+  - interface:
+      queryParameters:
+        if:
+          enum: ["oic.if.create"]
+
+
+
+/CreateResURI?if=oic.if.create:
+    description: |
+      Create an item using POST with oic.if.create, URI in the query parameter
+      Creates a new resource in the collection and a link pointing to it
+    displayName: create
+
+    post:
+      description: |
+        Create interface example
+
+      queryParameters:
+        href:
+          type: string
+
+      body:
+        application/json:
+          schema: create
+            example: |
+              [
+                {
+                  "rt": "oic.r.switch.binary",
```

Comment:
Should this be an array?  There may be a case where the created Resource has more than one "rt" value.